src/components/RecipeForm/IngredientsForm.test.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { render, fireEvent } from '@testing-library/react'
+import IngredientsForm from "./IngredientsForm";
+import { Ingredient } from "../../data/types";
+
+describe('<IngredientsForm>', () => {
+  beforeEach(() => {
```

Comment:
I've seen you are executing the same initialization for every test, jest provides this https://create-react-app.dev/docs/running-tests/#initializing-test-environment

we're using it in web